packages/design-system/package.json

```diff
@@ -132,6 +132,8 @@
     "@react-aria/textfield": "^3.8.1",
     "@react-stately/radio": "^3.6.2",
     "@react-stately/toggle": "^3.4.4",
+    "@xstyled/styled-components": "^3.8.0",
```

Comment:
There are some doubts about using another CSS-in-JS library. This will inflate the bundle, it is also not very clear what to use in the end for styles. Do you plan to switch to this library?